### ISSUE TITLE
Optimize filters on a shared report

### ISSUE THREAD

**Issue by mesellings** [camunda/camunda-docs#4528]
### Issue:
Feedback received on [View mode | Camunda 8 Docs](https://docs.camunda.io/optimize/components/userguide/view-mode/).

- Confusion about the expected behaviour of Optimize Filters on a Shared report. 
- [We include the following statement about shared report interactivity within the "Embedding:" section](https://docs.camunda.io/optimize/components/userguide/view-mode/), which is missed if you are not embedding:

> The shared versions of the dashboard allow only to view the dashboard itself. There is no possibility to alter it or interact with any other features of Optimize. Revoke the sharing anytime by disabling the share switch.

### Suggestion:
Perhaps this could be a separate statement at the end of those bullet points to make it more visible?

**Comment by akeller** [comment]
@mesellings this looks like a quick win, would you agree? If not, or if you would require help from the Optimize team, let's add it to their project board as well.

**Comment by mesellings** [comment]
Yep, this is straightforward, I can just look at this and get the Optimize team to review 👍
